bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx61.html

```diff
@@ -0,0 +1,161 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import google_play_button, send_to_device with context %}
+
+{% add_lang_files "firefox/whatsnew_59" "firefox/mobile" "firefox/sendto" %}
```

Comment:
Do we need all of these lang files? We at least need sendto but maybe we can eliminate whatsnew_59 and mobile?

Reply:
We can eliminate whatsnew_59 for sure. I am wondering whether we should keep mobile though.

Reply:
@craigcook, the strings look good, but I only see a handful of strings in the file on the WNP 61 test link. Are there other pages like sendto page to verify the strings?

Reply:
The page appears in several different states, which can be hard to simulate in a browser. But I can make screenshots of each state, that might be our best option. Is there an l10n bug I can attach them to?